install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -710,6 +710,25 @@ data:
   enable-egress-gateway: "true"
 {{- end }}
 
+{{- if .Values.externalVxlan.enabled }}
+  enable-external-vxlan: "true"
+{{- end }}
+
+{{- if .Values.vxlanEndpoint }}
+  # A space separated list of external VXLAN tunnel device endpoint IPs
+  external-vxlan-endpoint: {{ .Values.vxlanEndpoint | quote }}
+{{- end }}
+
+{{- if .Values.vxlanCIDR }}
+  # A space separated list of external VXLAN tunnel device network CIDRs
+  external-vxlan-cidr: {{ .Values.vxlanCIDR | quote }}
+{{- end }}
+
+{{- if .Values.vxlanMAC }}
+  # A space separated list of external VXLAN tunnel device MAC addresses (VTEP MAC)
+  external-vxlan-mac: {{ .Values.vxlanMAC | quote }}
+{{- end }}
```

Would it make sense to have the whole block in the `if .Values.externalVxlan.enabled` block?

```suggestion
{{- if .Values.externalVxlan.enabled }}
  enable-external-vxlan: "true"

{{- if .Values.vxlanEndpoint }}
  # A space separated list of external VXLAN tunnel device endpoint IPs
  external-vxlan-endpoint: {{ .Values.vxlanEndpoint | quote }}
{{- end }}

{{- if .Values.vxlanCIDR }}
  # A space separated list of external VXLAN tunnel device network CIDRs
  external-vxlan-cidr: {{ .Values.vxlanCIDR | quote }}
{{- end }}

{{- if .Values.vxlanMAC }}
  # A space separated list of external VXLAN tunnel device MAC addresses (VTEP MAC)
  external-vxlan-mac: {{ .Values.vxlanMAC | quote }}
{{- end }}
{{- end }}
```